### PR TITLE
fix table quirks

### PR DESCRIPTION
**Pull request by Mulesoft-Mysti**
a combination of factors required me to pull this out of tables and into lists. Did a little cleaning up while I was in there.

### CODE REVIEW

**Comment by Mulesoft-Mysti** [comment]
Adding Valkyrie since Fer is busy.

File: modules/ROOT/pages/munit-matchers.adoc [hunk 1]
```diff
@@ -5,251 +5,372 @@ endif::[]
...
-| Checks that the expression has a key that matches the specified matcher.
-
-[NOTE]
+Matchers are grouped according to the type of conditions you want to validate.
+
+== Match Core Values
+
+Core matchers evaluate core values in the expression:
+
+* Is the expression is null?
```
Comment (valkyrie69) [inline modules/ROOT/pages/munit-matchers.adoc]: Is the expression `null`

(remove extra is)

File: modules/ROOT/pages/munit-matchers.adoc [hunk 2]
```diff
@@ -5,251 +5,372 @@ endif::[]
...
+* What is the media Ttpe or encoding of the expression.
```
Comment (valkyrie69) [inline modules/ROOT/pages/munit-matchers.adoc]: Is this supposed to be media type?

File: modules/ROOT/pages/munit-matchers.adoc [hunk 3]
```diff
@@ -5,251 +5,372 @@ endif::[]
...
+* xref:core-matchers-reference.adoc#anyof-matchers[anyOf(Matchers[\])] checks if any of the matchers are successful.
++
+Example:
++
+[source,DataWeave,options="nowrap"]
+----
+#[MunitTools::anyOf(MunitTools::notNullValue(),MunitTools::withMediaType('text/xml'),MunitTools::isEmptyString())]
+----
+
+* xref:core-matchers-reference.adoc#allof-matchers[allOf(Matchers[\])] Checks if all of the matchers are successful.
```
Comment (valkyrie69) [inline modules/ROOT/pages/munit-matchers.adoc]: In the others you have lowercase checks in this one uppercase Checks

File: modules/ROOT/pages/munit-matchers.adoc [hunk 4]
```diff
@@ -5,251 +5,372 @@ endif::[]
...
+* xref:string-matchers-reference.adoc#equaltoignoringcase-string[equalToIgnoringCase(String)] checks that the expression is equal to the specified String, ignoring case.
++
+Example:
++
+[source,DataWeave,options="nowrap"]
+----
+#[MunitTools::equalToIgnoringCase('Example')]
+----
+
+* xref:string-matchers-reference.adoc#equaltoignoringwhitespace-string[equalToIgnoringWhiteSpace(String)] checks that the expression is equal to the string disregarding leading and trailing white spaces, and compression all inner white spaces to a single space.
```
Comment (valkyrie69) [inline modules/ROOT/pages/munit-matchers.adoc]: Not sure about this one.  

checks that the expression is equal to the string, disregarding leading and trailing white spaces, and **compressing** all inner white spaces to a single space. 

?

File: modules/ROOT/pages/munit-matchers.adoc [hunk 5]
```diff
@@ -5,251 +5,372 @@ endif::[]
...
+[source,DataWeave,options="nowrap"]
+----
+#[MunitTools::stringContainsInOrder('an', 'example')]
+----
+
+== Match Comparable Expressions
+
+Comparable matchers compare the expression against a specified value:
+
+* Is the expression's value greater or smaller than a specified value?
```
Comment (valkyrie69) [inline modules/ROOT/pages/munit-matchers.adoc]: Greater or smaller
OR 
Greater or less than
?

File: modules/ROOT/pages/munit-matchers.adoc [hunk 6]
```diff
@@ -5,251 +5,372 @@ endif::[]
...
+----
+#[MunitTools::equalTo({example1: 1 , example2: 2}]
+----
+
+== Match Maps and Iterations
+
+Use iterable matchers and maps matchers when your expression is an array or a map of data:
+
+* Is the map or array in the expression empty?
+* Does the map or array in the expression have a specific key.
```
Comment (valkyrie69) [inline modules/ROOT/pages/munit-matchers.adoc]: All the other bullet points have a ? at the end

File: modules/ROOT/pages/munit-matchers.adoc [hunk 7]
```diff
@@ -5,251 +5,372 @@ endif::[]
...
+* Is every item in the map or array from the expression greater than or less than a specific number?
+* Does any item in the map or array from the expression start or end with a specified string?
+
+Iterable and Map Matchers:
+
+* xref:iterable-map-matchers-reference.adoc#everyitem-matcher[everyItem(Matcher)] checks that every element in the expression matches the specified matcher. This matcher only works for Arrays.
```
Comment (valkyrie69) [inline modules/ROOT/pages/munit-matchers.adoc]: This matcher works only for Arrays.

File: modules/ROOT/pages/munit-matchers.adoc [hunk 8]
```diff
@@ -5,251 +5,372 @@ endif::[]
...
+----
+#[MunitTools::everyItem(MunitTools::notNullValue())]
+----
++
+[source,DataWeave,options="nowrap"]
+----
+#[MunitTools::everyItem(MunitTools::startsWith('a'))]
+----
+
+* xref:iterable-map-matchers-reference.adoc#hasitem-matcher[hasItem(Matcher)] checks that any element in the expression matches the specified matcher. This matcher only works for Arrays.
```
Comment (valkyrie69) [inline modules/ROOT/pages/munit-matchers.adoc]: ...works only for...

File: modules/ROOT/pages/munit-matchers.adoc [hunk 9]
```diff
@@ -5,251 +5,372 @@ endif::[]
...
+* xref:iterable-map-matchers-reference.adoc#isempty[isEmpty()] checks that the expression is an empty collection.
++
+Examples:
++
+[source,DataWeave,options="nowrap"]
+----
+#[MunitTools::isEmpty()]
+----
+
+* xref:iterable-map-matchers-reference.adoc#haskey-matcher[hasKey(Matcher)] checks that the expression has a key that matches the specified matcher. This matcher only works for Maps.
```
Comment (valkyrie69) [inline modules/ROOT/pages/munit-matchers.adoc]: ...works only for...

Throughout--I won't comment on any more. It's a minor thing anyway.

**Review by valkyrie69 — APPROVED** [review]
Minor suggestions. No show stoppers. Looks a lot better!

File: modules/ROOT/pages/munit-matchers.adoc (hunk 4 above)
Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/munit-matchers.adoc]: yup!

File: modules/ROOT/pages/munit-matchers.adoc (hunk 5 above)
Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/munit-matchers.adoc]: i went with bigger or smaller than

**Pull request merged**